mpisppy/utils/sputils.py

```diff
@@ -130,7 +130,14 @@ def spin_the_wheel(hub_dict, list_of_spoke_dict, comm_world=None):
 
     return spcomm, opt_dict
 
-def first_stage_solution_writer( file_name, scenario, bundling ):
+def first_stage_nonant_npy(file_name, scenario, bundling):
```

Comment:
I'd be happier with this function name if it had `writer` or `serializer` or something similar in the name

Reply:
OK. I will replace npy with npy_serializer